synapse/models/risk.py

```diff
@@ -1,14 +1,43 @@
+import synapse.exc as s_exc
+
+import synapse.lib.chop as s_chop
+import synapse.lib.types as s_types
 import synapse.lib.module as s_module
 
-cvss_v2_re = r'AV:(L|A|N)\/AC:(H|M|L)\/Au:(M|S|N)\/C:(N|P|C)\/I:(N|P|C)\/A:(N|P|C)(\/E:(ND|U|POC|F|H)\/RL:(ND|OF|TF|W|U)\/RC:(ND|UC|UR|C))?(\/CDP:(ND|N|L|LM|MH|H)\/TD:(ND|N|L|M|H)\/CR:(ND|L|M|H)\/IR:(ND|L|M|H)\/AR:(ND|L|M|H))?'
 
-cvss_v3_re = r'AV:(N|A|L|P)\/AC:(L|H)\/PR:(N|L|H)\/UI:(N|R)\/S:(U|C)\/C:(N|L|H)\/I:(N|L|H)\/A:(N|L|H)(\/E:(X|U|P|F|H)\/RL:(X|O|T|W|U)\/RC:(X|U|R|C))?(\/CR:(X|L|M|H)\/IR:(X|L|M|H)\/AR:(X|L|M|H)\/MAV:(X|N|A|L|P)\/MAC:(X|L|H)\/MPR:(X|N|L|H)\/MUI:(X|N|R)\/MS:(X|U|C)\/MC:(X|N|L|H)\/MI:(X|N|L|H)\/MA:(X|N|L|H))?'
+class CvssV2(s_types.Str):
+
+    def _normPyStr(self, text):
+        try:
+            return s_chop.cvss2_normalize(text), {}
+
+        except s_exc.BadDataValu as exc:
+            mesg = exc.get('mesg')
+            raise s_exc.BadTypeValu(mesg=mesg) from None
```

Comment:
can we add `name=self.name` so we capture the name of the property/form as well?

Reply:
I removed this and changed the BadDataValu's to BadTypeValu in chop